pages/backgrounds/knave.md

```diff
@@ -56,14 +56,38 @@ You specialised in a particular criminal enterprise. Choose a speciality and gai
 
 You gain the ability to climb twice as fast. When you make a running jump double the distance you can cover.
 
+<header>
+
+## Cutpurse
+
+You are a pickpocket. Your prey is the unwary, your bread and butter the coins in their purses.
+
+<p class="subheading">Speciality</p>
+
+</header>
+
+| Level             | Features    |
+| ----------------- | - |
+| 1st               | Proficient: Sleight, Pilfer |
+
+<header>
+
+### Pilfer
+
+<p class="subheading">1st-level Pickpocket speciality feature</p>
+
+</header>
+
+Your hands have a mind of their own, regularly pocketing random items and stowing them. You have a **d6 Pouch** that can contain any **Small Equipment Item**. This can **Renew** your pouch when you rest in a **Settlement**.
+
 ## Ragamuffin
 
 You grew up running the city streets, playing games, stealing food, dodging the watch, scrapping and generally running amok. You survived, despite the odds, through your wits and by making yourself useful to the local criminal elements.
 
 | Level             | Speciality Features    |
 | ----------------- | - |
-| 1st               | Proficient: Deception, Street Wise |
+| 1st               | Proficient: Deception, Slippery |
 
-### Street Wise
+### Slippery
 
-You can add double your proficiency bonus to any [Wisdom](pages/characters/attributes/wisdom.md) [check](pages/rules/rolling?id=checks) related to cities.
+You have advantage on checks to escape bonds such as manacles, or nets. Additionally, you have advantage on checks to evade being followed or tracked.
```

Comment:
We shouldn't avoid using advantage as a feature benefit. Advantage/disadvantage should only be available through fictional positioning.